composer/algorithms/factorize/factorize.py

```diff
@@ -44,9 +45,9 @@ def _maybe_replace_conv2d(module: torch.nn.Module, module_index: int) -> Optiona
             return FactorizedConv2d.from_conv2d(module, module_index, latent_channels=latent_channels)
         return None  # not enough rank reduction to be worth it
 
-    ret = surgery.replace_module_classes(model,
-                                         optimizers=optimizers,
-                                         policies={torch.nn.Conv2d: _maybe_replace_conv2d})
+    ret = module_surgery.replace_module_classes(model,
+                                                optimizers=optimizers,
```

Comment:
Is this extra whitespace from tabs that should be removed? Can't really tell

Reply:
I think it's from renaming `surgery` to `module_surgery`